### PR TITLE
fix: don't remove all stars from the docs

### PR DESCRIPTION
**Pull request by sigmaSd**
fix https://github.com/denoland/deno_doc/issues/523

### CODE REVIEW

**Comment by sigmaSd** [comment]
I noticed that some correct stars still gets stripped like `**Example**`, not sure whats the best way to handle these

**Comment by crowlKats** [comment]
@sigmaSd can you add some tests?

**Comment by sigmaSd** [comment]
I added some tests

File: src/util/swc.rs [hunk 1]
```diff
@@ -124,3 +126,64 @@ pub fn module_export_name_value(
...
+ * # Program
+ * **Example:**
+ *
+ * example1
+ */
+"#
+      ),
+      r#"/**
+# Program
+**Example:**
```
Comment (crowlKats) [inline src/util/swc.rs:183]: this doesnt seem to be getting stripped, unlike what you mentioned in your previous comment

Comment (sigmaSd) [reply src/util/swc.rs:183]: Yes what I had is 
```
/**
** Example** // no starting star 
*/
```

But then I guess maybe we can consider this a user error to not use the star at the start

**Review by crowlKats — APPROVED** [review]
LGTM

**Pull request merged**